src/VisualStudio/CSharp/Impl/Options/Formatting/NewLinesViewModel.cs

```diff
@@ -206,6 +206,7 @@ public NewLinesViewModel(OptionStore optionStore, IServiceProvider serviceProvid
             Items.Add(new CheckBoxOptionViewModel(CSharpFormattingOptions.NewLinesForBracesInMethods, CSharpVSResources.Place_open_brace_on_new_line_for_methods_local_functions, s_methodPreview, this, optionStore));
             Items.Add(new CheckBoxOptionViewModel(CSharpFormattingOptions.NewLinesForBracesInProperties, CSharpVSResources.Place_open_brace_on_new_line_for_properties_indexers_and_events, s_propertyPreview, this, optionStore));
             Items.Add(new CheckBoxOptionViewModel(CSharpFormattingOptions.NewLinesForBracesInAccessors, CSharpVSResources.Place_open_brace_on_new_line_for_property_indexer_and_event_accessors, s_propertyPreview, this, optionStore));
+            Items.Add(new CheckBoxOptionViewModel(CSharpFormattingOptions.NewLinesForBracesInPropertyPatternClauses, CSharpVSResources.Place_open_brace_on_new_line_for_property_pattern_clauses, s_propertyPreview, this, optionStore));
```

Comment:
`s_propertyPreview` isn't correct here. You'll need to add a new constant `s_popertyPatternPreview`.